app/models/user.rb

```diff
@@ -3,4 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  has_one :homepage, dependent: :destroy
+  after_create -> { self.create_homepage }
```

Comment:
This makes me nervous. We're depending on a homepage to exist, but due to testing, it might not. I'm not entirely sure how I want to handle this.